components/Common/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import { type LinkProps } from 'next/link';
+import { useMemo, type FC } from 'react';
+
+import LocalizedLink from '@/components/LocalizedLink';
+
+import BreadcrumbItem from './BreadcrumbItem';
+import BreadcrumbRoot from './BreadcrumbRoot';
+import styles from './index.module.css';
+
+type BreadcrumbLink = {
+  label: string;
+  href: LinkProps['href'];
+};
+
+type BreadcrumbsProps = {
+  links: BreadcrumbLink[];
+  maxLength?: number;
+  hideHome?: boolean;
+};
+
+const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  links,
+  maxLength = 3,
+  hideHome = false,
+}) => {
+  const items = useMemo(
+    () =>
+      (links.length > maxLength
+        ? links.slice(links.length - maxLength)
+        : links
+      ).map((link, index, items) => {
+        const isLastItem = index === items.length - 1;
+        return (
+          <BreadcrumbItem key={link.href.toString()} hideSeparator={isLastItem}>
+            <LocalizedLink
+              href={link.href}
+              className={classNames({
+                [styles.active]: isLastItem,
+              })}
+              aria-current={isLastItem ? 'page' : undefined}
```

Comment:
```suggestion
              aria-current={isLastItem && 'page'}
```

Could you check if this works? (I fear that it would render a string `false`

Reply:
Yup, it would render a string `false`.

Reply:
Yup, it will render a string with value `false`.